packages/RNSocialX/src/components/interaction/MediaInteractionButtons.style.ts

```diff
@@ -1,3 +1,4 @@
+import { Row } from 'native-base';
```

Comment:
?

Reply:
I think Ionut imported it, I'll look into it